prisma/migrations/20240226024040_init/migration.sql

```diff
@@ -0,0 +1,93 @@
+-- CreateEnum
+CREATE TYPE "preeshHeaviness" AS ENUM ('LOWKEY', 'HEAVY', 'HEAVYFR', 'SUPERPREESH');
+
+-- CreateTable
+CREATE TABLE "Beast" (
+    "id" SERIAL NOT NULL,
+    "gamerTag" VARCHAR(20) NOT NULL,
+    "email" TEXT NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" TIMESTAMP(3) NOT NULL,
+
+    CONSTRAINT "Beast_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "Preesh" (
+    "id" SERIAL NOT NULL,
+    "text" TEXT NOT NULL,
+    "authorId" INTEGER NOT NULL,
+    "receiverId" INTEGER NOT NULL,
+    "praysh" BOOLEAN NOT NULL DEFAULT false,
+    "heaviness" "preeshHeaviness" NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" TIMESTAMP(3) NOT NULL,
```

Comment:
updatedAt should update automatically. Think you can do below:

`@default(dbgenerated("CURRENT_TIMESTAMP(3) ON UPDATE CURRENT_TIMESTAMP(3)"))`

Found this quickly on another repo -- would confirm it works first. Also updatedAt should include a default value too